examples/05-Q3D/Q2D_Armoured_Cable.py

```diff
@@ -0,0 +1,344 @@
+"""
+Q2D: Cable parameter identification
+---------------------------------------------------
+This example shows how you can use PyAEDT to perform these tasks:
+
+ - Create a Q2D design using the Modeler primitives and importing part of the geometry.
+ - Set up the entire simulation.
+ - Link the solution to a Simplorer design.
+
+ For cable information, see `4 Core Armoured Power Cable <https://www.luxingcable.com/low-voltage-cables/4-core-armoured-power-cable.html>`_
+
+"""
+#################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+
+import pyaedt
+import math
+
+#################################################################################
+# Initialize core strand dimensions and positions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize cable sizing - radii in mm
```

Comment:
```suggestion
# Initialize cable sizing - radii in mm.
```
The idea is to keep the titles short so that the display text in the left column of the doc's navigation pane isn't too long. The sentence following a title can then provide a more detailed description. Later in this example, there are some long titles and no descriptive sentences after the ttitles.